app/app.go

```diff
@@ -311,12 +312,18 @@ func init() {
 }
 
 // GetWasmOpts build wasm options
-func GetWasmOpts(appOpts servertypes.AppOptions) []wasm.Option {
+func GetWasmOpts(nibiru *NibiruApp, appOpts servertypes.AppOptions) []wasm.Option {
```

Comment:
Since this param to the bottom of the call stack use type, not pointer, as you are doing a reference/de-reference dance. Using always the type and not the pointer will make it more consistent.

Reply:
done